proxy/core/src/main/scala/io/cloudstate/proxy/eventing/GooglePubsubEventing.scala

```diff
@@ -106,22 +99,22 @@ final class PubSubSettings private (
   def createClientSettings()(implicit sys: ActorSystem): GrpcClientSettings = {
     val sslConfig = rootCa.fold("") { rootCa =>
       s"""
-      |ssl-config {
-      |  disabledKeyAlgorithms = []
-      |  trustManager = {
-      |    stores = [
-      |      { type = "PEM", path = "$rootCa", classpath = true }
-      |    ]
-      |  }
-      |}""".stripMargin
+         |ssl-config {
+         |  disabledKeyAlgorithms = []
+         |  trustManager = {
+         |    stores = [
+         |      { type = "PEM", path = "$rootCa", classpath = true }
+         |    ]
+         |  }
+         |}""".stripMargin
```

Comment:
Hmm, I did some cut and pasting, I can put this all back the way it was before the merge.